atc/api/cli_test.go

```diff
@@ -62,6 +64,12 @@ var _ = Describe("CLI Downloads API", func() {
 
 		err = windowsArchive.WriteZip(zipFile)
 		Expect(err).NotTo(HaveOccurred())
+
+		fakeaccess = new(accessorfakes.FakeAccess)
+	})
+
+	JustBeforeEach(func() {
+		fakeAccessor.CreateReturns(fakeaccess)
```

Comment:
If this setup needs to happen in a few different tests it can probably be moved into the [api_suite_test.go](https://github.com/concourse/concourse/blob/master/atc/api/api_suite_test.go), but it's not a big deal.

Reply:
I can probably convert that over if you would like before things get merged in.  Let me know if you want me to tackle that on my end. or if you folks want to do it on your end.

Reply:
I went ahead and consolidated those into [api_suite_test.go](https://github.com/loghen41/concourse/blob/fded134857203080926b9b166e913802205f0a2d/atc/api/api_suite_test.go#L208) and removed them from the cli_test, builds_test and info_test.

You should be hopefully good to go. Unit tests passed on my end.I I'll resolve this conversation provided they pass on your end as well.